src/components/Button.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text, StyleSheet, Pressable } from 'react-native';
+
+export default function Button (props) {
```

Comment:
It is better do not give same as ReactNative Components name. To avoid name collision.